python/lsst/ts/sal/make_idl_file.py

```diff
@@ -55,17 +55,6 @@ def __init__(self, name, keep_all):
         )
         self.idl_file_to_path = idl.get_idl_dir() / idl_file_name
 
-    def copy_xml_files(self):
-        interfaces_dir = os.path.join(self.xml_dir, "sal_interfaces")
-        std_xml_paths = [
-            os.path.join(interfaces_dir, f"{n}.xml")
-            for n in ("SALGenerics", "SALSubsystems")
-        ]
-        sal_xml_paths = glob.glob(os.path.join(interfaces_dir, self.name, "*.xml"))
-
-        for xmlpath in std_xml_paths + sal_xml_paths:
-            shutil.copy(xmlpath, self.sal_work_dir)
-
```

Comment:
I'm happy to see this isn't needed.

Should delete_files be updated? There's no point if ts_sal's salgenerator is copying the xml files. But if the xml files are being read in place, are there still xml files to delete at line 71?

Reply:
salgenerator will generate the <CSC>_Generics.xml in the $SAL_WORK_DIR and use the others from ts_xml directly